src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,95 @@
 package racingcar;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+class Car {
+	String name;
+	int step;
+	String show;
+
+	public Car(String n) {
+		name = n;
+		step = 0;
+		show = "";
+	}
+
+	public void go() {
+		int randomNumber = Randoms.pickNumberInRange(1, 9);
+
+		if (randomNumber >= 4) {
+			step++;
+			show = show + "-";
+		}
+	}
+
+	public void print() {
+		System.out.printf("%s : %s\n", name, show);
+	}
+}
+
 public class Application {
-    public static void main(String[] args) {
-        // TODO: 프로그램 구현
-    }
+	public static void result(Car[] c, int N) {
+		for (int i = 0; i < N; i++) {
+			c[i].go();
+			c[i].print();
+		}
+	}
+
+	public static int findMax(Car[] c, int N) {
+		int maxStep = 0;
+		for (int i = 0; i < N; i++) {
+			if (c[i].step > maxStep) {
+				maxStep = c[i].step;
+			}
+		}
+		return maxStep;
+	}
+
+	public static String winner(Car[] c, int maxStep, int N) {
+		ArrayList<String> winners = new ArrayList<String>();
+		for (int i = 0; i < N; i++) {
+			if (maxStep == c[i].step) {
+				winners.add(c[i].name);
+			}
+		}
+		return String.join(", ", winners);
+	}
+
+	public static void main(String[] args) throws IllegalArgumentException {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+		String input = sc.nextLine();
+		System.out.println("시도할 회수는 몇회인가요?");
+		int trial = sc.nextInt();
+		sc.close();
+
+		ArrayList<String> carNames = new ArrayList<String>();
+		Collections.addAll(carNames, input.split(","));
+		int N = carNames.size();
+
+		for (int i = 0; i < N; i++) {
+			String tmp = carNames.get(i);
+			if (tmp.length() > 5) {
+				throw new IllegalArgumentException();
+			}
+		}
```

Comment:
1. 2주차 요구사항 중 하나가 `함수(또는 메서드)가 한 가지 일만 하도록 최대한 작게 만들어라.`인데요, 이 부분을 메서드로 분리할 수 있을 것 같습니다.

2. 반복문을 사용할 때 [for-each loop](https://docs.oracle.com/javase/8/docs/technotes/guides/language/foreach.html) 을 사용하는 것을 고려해보세요 ! 

3. `tmp` 와 같은 변수명보다는 `carName`과 같은 변수명을 통해 의도를 드러내면 좋겠습니다.

```suggestion
		for (String carName : carNames) {
			if (isInvalidCarName(carName)) {
				throw new IllegalArgumentException();
			}
		}
```